src/module/server-group/ServerGroupCtrl.ts

```diff
@@ -53,6 +53,26 @@ export class ServerGroupCtrl {
     return this.isServerInState(server, SERVER_STATE_RUNNING);
   }
 
+  isAtLeastOneServerInRunningState(): boolean {
+    return this.isAtLeastOneServerInServerGroupInState(SERVER_STATE_RUNNING);
+  }
+
+  isAtLeastOneServerInReloadRequiredState(): boolean {
+    return this.isAtLeastOneServerInServerGroupInState(SERVER_STATE_RELOAD_REQUIRED);
+  }
+
+  isAtLeastOneServerInRestartRequiredState(): boolean {
+    return this.isAtLeastOneServerInServerGroupInState(SERVER_STATE_RESTART_REQUIRED);
+  }
+
+  isAtLeastOneServerInStoppedState(): boolean {
+    return this.isAtLeastOneServerInServerGroupInState(SERVER_STATE_STOPPED);
+  }
+
+  areAllServersInStoppedState(): boolean {
+    return this.areAllServersInServerGroupInState(SERVER_STATE_STOPPED);
```

Comment:
@vblagoje All of these methods will result in a call to the server to get the status map array, so each page load will make the same call at least 5 times. It would be better if we pulled the status map once upon page load. When a user clicks stop/start/etc for a cluster, we should then force a page refresh (I think we already do this in most cases) and the user will be presented with the updated status